pwa/static/.env.development

```diff
@@ -0,0 +1,13 @@
+# Backend-config
+GATSBY_ME_URL=https://api.opencatalogi.nl/api/users/me
+GATSBY_API_URL=https://api.opencatalogi.nl/api
+GATSBY_ADMIN_URL=https://api.opencatalogi.nl/admin
+GATSBY_BASE_URL=https://api.opencatalogi.nl
+GATSBY_FRONTEND_URL=https://api.opencatalogi.nl
+GATSBY_ORGANIZATION=""

```

Comment:
I've always thought this organization key was relevant for the deployment. However, now that we're going to deploy using our own workflow I checked our usage of `GATSBY_ORGANIZATION` and cannot find any.

Might be safe to delete?